deploy/kubernetes/metricbeat/README.md

```diff
@@ -2,30 +2,145 @@
+- [metricbeat-daemonset.yaml](metricbeat-daemonset.yaml) to create a [DeamonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/). This way we ensure we get a running metricbeat daemon on each node of the cluster.
+  This files uses a set of environment variables to configure Elasticsearch output.
 
-Everything is deployed under `kube-system` namespace, you can change that by
-updating YAML manifests under this folder.
+| Variable               | Default       | Description                          |
+|------------------------|---------------|--------------------------------------|
+| ELASTICSEARCH_HOST     | elasticsearch | Elasticsearch host                   |
+| ELASTICSEARCH_PORT     | 9200          | Elasticsearch port                   |
+| ELASTICSEARCH_USERNAME | elastic       | Elasticsearch username for HTTP auth |
+| ELASTICSEARCH_PASSWORD | changeme      | Elasticsearch password               |
+| ELASTIC_CLOUD_ID       |               | Elastic cloud ID                     |
+| ELASTIC_CLOUD_AUTH     |               | Elastic cloud auth                   |
 
-### Settings
+- [metricbeat-role-binding.yaml](metricbeat-role-binding.yaml),[metricbeat-role.yaml](metricbeat-role.yaml) and [metricbeat-service-account.yaml](metricbeat-service-account.yaml) to define a user's permissions in our namespace (more information on this can be found [here](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)). Notice that the namespace we are using is `kube-system`, but this can be changed by updating the YAML manifests.
+
+### Example
+
+In this example, we will use `kind` to run a local Kubernetes cluster. Our cluster will have more than one node.
+This way we can see the usage of our Metricbeat DaemonSet.
+
+
+#### Prerequisites
+
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/) to run commands against Kubernetes clusters.
+- [For option 1](#Option-1.-Run-locally-on-kind): [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/) for running local Kubernetes clusters.
+- [For option 2](#Option-2.-Run-on-GKE): [gcloud Cli](https://cloud.google.com/sdk/docs/install).
+
+#### Option 1. Run locally on kind
+
+The first thing we need to do is create a Kubernetes cluster. On kind, the default [configuration](https://kind.sigs.k8s.io/docs/user/configuration/) of a cluster generates only one node. As we are using a DaemonSet to run our Metricbeat, we will make a cluster run 4 nodes, so we can check our pods are running on all (or some) of them. To do this, we simply create the file `cluster-config.yaml`:
+```YAML
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+- role: worker
+- role: worker
+- role: worker
+```
+
+After that, we can create a cluster using the command below.
+
+```
+$ kind create cluster --config=cluster-config.yaml
```

Comment:
I usually tend to fix the k8s cluster by passing a specific version with `--image=kindest/node:v1.23.5`